i18n/zh/docusaurus-plugin-content-docs/current/user-guide/administration/performance-tuning-tips.md

```diff
@@ -29,8 +29,8 @@ GreptimeDB 提供了各种指标来帮助监控和排查性能问题。官方仓
 > 提示: 从 v0.11 版本开始，在使用远程对象存储服务时，系统会默认启用本地缓存（包括读取和写入）。通常情况下，您只需要根据需求调整缓存容量即可。
 
 读取缓存将对象或一段范围的数据存储在本地磁盘上，以避免再次从远程读取相同的数据。以下示例展示了如何为 S3 启用读取缓存。
-- `cache_path` 是存储缓存对象的目录，从 v0.11 版本开始默认值为 `{data_home}/object_cache/read` 目录。
-- `cache_capacity` 是缓存的容量。从 0.11 版本开始，默认初始值为 `5Gib`，建议至少留出总磁盘空间的 1/10 用于缓存。
+- `cache_path` 是存储缓存对象的目录，从 v0.11 版本开始不再需要手动设置。
+- `cache_capacity` 是缓存的容量。从 0.11 版本开始，默认初始值为 `5GiB`，建议至少留出总磁盘空间的 1/10 用于缓存。
```

Comment:
I believe a ratio of 1/10 is too conservative; 1/3 or 4/1 would be more appropriate.

Reply:
This is a minimum value. We may also have write cache, WAL, and logs on disk.